blueprints/frontend-shared-config/files/.jscsrc

```diff
@@ -9,11 +9,14 @@
   "fileExtensions": [".js"],
   "esnext": true,
   "disallowEmptyBlocks": true,
+  "disallowMultipleSpaces": true,
```

Comment:
I'm not sure how this works, but do we need to add a file for new rules in the `/blueprints/frontend-shared-config/files/lib/jscs-rules/` directory? Or are they looked up elsewhere?